bc_obps/reporting/json_schemas/2024/gsc_non_compression_non_combustion/with_useful_energy.json

```
@@ -0,0 +1,76 @@
+{
+  "type": "object",
+  "title": "General stationary combustion of fuel or waste with production of useful energy",
```

```suggestion
  "title": "General stationary combustion of fuel or waste at a linear facilities operation resulting in the production of useful energy",
```
The LFO GSC activities have similar, but different source types